apps/gepa-ontology-workbench/app/globals.css

```
@@ -0,0 +1,54 @@
+  --muted: #a8b0aa;
+  --surface: #202728;
+  --focus: #fff3b3;
+}
+
+* { box-sizing: border-box; }
+html { background: var(--mineral-deep); }
+body { margin: 0; background: var(--mineral); color: var(--ivory); font-family: Arial, Helvetica, sans-serif; }
+button, input { font: inherit; }
+button { color: inherit; cursor: pointer; }
+button:focus-visible, input:focus-visible, a:focus-visible { outline: 3px solid var(--focus); outline-offset: 3px; }
+.sr-only { position: absolute; width: 1px; height: 1px; overflow: hidden; clip: rect(0, 0, 0, 0); white-space: nowrap; }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)